docs/data-ai/data/data-pipelines.md

```diff
@@ -0,0 +1,756 @@
+
+```go
+client, err := utils.NewViamClient(context.Background(), utils.WithAPIKey("<api-key>", "<api-key-id>"))
+if err != nil {
+    panic(err)
+}
+defer client.Close()
+
+dataClient := client.DataClient()
+
+_, err := dataClient.DisableDataPipeline(context.Background(), &datapb.DisableDataPipelineRequest{
+    Id: "<pipeline-id>",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.DisableDataPipeline`](/dev/reference/apis/data-client/#disabledatapipeline) to disable a data pipeline:
+
+```typescript
+const apiKey = "<api-key>";
+const apiKeyID = "<api-key-id>";
+
+const client = await createViamClient({
+  credential: {
+    type: "api-key",
+    payload: { key: apiKey, keyId: apiKeyID },
+  },
+});
+
+const dataClient = client.dataClient;
+
+await dataClient.disableDataPipeline({ id: "<pipeline-id>" });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Delete a pipeline
+
+{{< tabs >}}
+{{% tab name="CLI" %}}
+
+Use [`datapipelines delete`](/dev/tools/cli/#datapipelines) to delete a data pipeline:
+
+```sh {class="command-line" data-prompt="$"}
+viam datapipelines delete --id=<pipeline-id>
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+Use [`DataClient.DeleteDataPipeline`](/dev/reference/apis/data-client/#deletedatapipeline) to delete a data pipeline:
+
+```python
+data_client = DataClient.from_api_key(
+    api_key="<api-key>",
+    api_key_id="<api-key-id>"
+)
+
+data_client.delete_data_pipeline(id="<pipeline-id>")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+Use [`DataClient.DeleteDataPipeline`](https://pkg.go.dev/go.viam.com/rdk/app#DataClient.DeleteDataPipeline) to delete a data pipeline:
+
+```go
+client, err := utils.NewViamClient(context.Background(), utils.WithAPIKey("<api-key>", "<api-key-id>"))
+if err != nil {
+    panic(err)
+}
+defer client.Close()
+
+dataClient := client.DataClient()
+
+_, err := dataClient.DeleteDataPipeline(context.Background(), &datapb.DeleteDataPipelineRequest{
+    Id: "<pipeline-id>",
+})
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+Use [`dataClient.DeleteDataPipeline`](/dev/reference/apis/data-client/#deletedatapipeline) to delete a data pipeline:
+
+```typescript
+const apiKey = "<api-key>";
+const apiKeyID = "<api-key-id>";
+
+const client = await createViamClient({
+  credential: {
+    type: "api-key",
+    payload: { key: apiKey, keyId: apiKeyID },
+  },
+});
+
+const dataClient = client.dataClient;
+
+await dataClient.deleteDataPipeline({ id: "<pipeline-id>" });
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Check pipeline execution history
+
+Data pipeline executions may have any of the following statuses:
+
+- `SCHEDULED`: pending execution
+- `STARTED`: currently processing
+- `COMPLETED`: successfully finished
+- `FAILED`: execution error
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Use [`DataClient.ListDataPipelineRuns`](/dev/reference/apis/data-client/#listdatapipelineruns) to view the statuses of past executions of a pipeline:
```

Comment:
I wouldn't say past. A run might not be complete yet. Also like I mentioned elsewhere, shows more than status